app/components/message_attachments/message_attachment.js

```diff
@@ -113,9 +113,9 @@ export default class MessageAttachment extends PureComponent {
         });
 
         return (
-            <View style={style.bodyContainer}>
+            <React.Fragment>
```

Comment:
why not `return content`;  I don't think you need the fragment here if you no longer need the *View*